CHANGELOG.md

```diff
@@ -21,6 +21,54 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 -->
 
+## [2.0.0-rc.3] - 2025-10-29
+
+### Added
+
++ The latest version change in licensing terms from MIT to a [dual license model of AGPL 3.0 and commercial](https://flamegpu.com/download/license/). This requires user contributions to sign our CLA ([#1313](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1313))
++ Added support for Python 3.14 ([#1320](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1320))
++ Added CMake 4 support ([#1315](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1315))
++ CUDA 13 Support added ([#1302](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1302))
++ Migration to Jitify2 adding significant RTC kernel compilation speed-ups (using patch until [#146](https://github.com/NVIDIA/jitify/pull/146) is merged) ([#1150](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1150))
++ Added CUDA 12.8 and 12.9 to known CUDA downloads for windows ([#1296](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1296))
++ Added HostEnvironmentDirectedGraph::Iterator to allow vertex and edge iteration ([#183](https://github.com/FLAMEGPU/FLAMEGPU2-docs/pull/183))
+
+### Changed (Breaking)
+
++ Added `nlohmann::json` (replacing RapidJSON) with some breaking changes for nan/inf ([#1277](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1277))
+
+### Changed
+
++ CI Windows CUDA install method updated to use faster redist packages ([#1314](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1314))
++ CI Windows builds now have a timeout to catch occasional hanging of workflow ([#1305](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1305))
++ CI Draft release workflow installs `setuptools` and `wheel` for all Python versions ([1306](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1306)) 
++ Circles example logging has been updated to truncate log files ([#1301](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1301))
++ Switched to TelemetryDeck v2.0 API ([#1300](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1300))
++ Improved CMake warnings for unsupported CUDA versions ([#1296](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1296))
++ Changed default use of `patchelf` on linux to off ([#1299](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1299))
++ Improved device exception message for bucket messaging ([#1298](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1298))
++ Upgrade to CCCL 2.3.2 support ([#1155](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1155))
++ CI updated GitHub actions to Ubuntu 24.04 Workflows ([#1295](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1295))
++ CI Updated support for ManyLinux 2_28 and removal of ManyLinux2014 from GitHub actions ([#1228](https://github.com/FLAMEGPU/FLAMEGPU2/pull/1228))
```

Comment:
I'd consider this a breaking change (requires user action for some users), that pyflamegpu wheels from CI now reuiqre glibc >= 2.28, but could be left here.